BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.m

```diff
@@ -289,6 +290,16 @@ - (void)start
     [[BOXAPIOperation APIOperationGlobalLock] unlock];
 }
 
+- (BOOL)shouldUseSessionTask
+{
+    return NO;
+}
+
+- (NSURLSessionTask *)sessionTask
+{
+    return nil;
```

Comment:
sessionTask will be created by the subclasses accordingly. I'll put a comment in.